x-pack/plugins/observability_ai_assistant/public/components/action_menu_item/action_menu_item.tsx

```diff
@@ -30,6 +30,20 @@ export function ObservabilityAIAssistantActionMenuItem() {
 
   const initialMessages = useMemo(() => [], []);
 
+  useEffect(() => {
+    const keyboardListener = (event: KeyboardEvent) => {
+      if (event.ctrlKey && event.code === 'Semicolon') {
```

Comment:
Is this documented anywhere for users ?

Reply:
Not yet. Perhaps we can add something instead of the friendly Moose image. @boriskirov ?

Reply:
What about a tooltip on the AI Assistant button that describes the shortcut?

<img width="324" alt="image" src="https://github.com/elastic/kibana/assets/209966/4fb726a8-18a8-4741-bdc7-5745e0392ee5">


Similar to what you often is in Desktop menus:

<img width="317" alt="image" src="https://github.com/elastic/kibana/assets/209966/7529a564-d8f7-4134-838c-670aa8bb0691">

Reply:
I like it! @boriskirov wdyt?

Reply:
Yes, good idea!